### DISCUSSION TITLE
pnpm bun support without node

### DISCUSSION THREAD

**Discussion by lorypelli** [pnpm/pnpm#8454] [Q&A]
Does pnpm work with bun without having node installed on the system? If not, you should make so...

**Comment by zkochan** [comment]
pnpm works without node.js because we bundle pnpm with node.js itself.

**Reply by lorypelli** [reply]
> pnpm works without node.js because we bundle pnpm with node.js itself.

what if I do `bun add -g pnpm` to install it globally with bun without having node, does it install node?
